precompiles/bank/integration_test.go

```diff
@@ -0,0 +1,122 @@
+package bank_test
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/evmos/v15/precompiles/bank"
+
+	"github.com/evmos/evmos/v15/precompiles/testutil"
+	"github.com/evmos/evmos/v15/testutil/integration/evmos/keyring"
+	//nolint:revive // dot imports are fine for Ginkgo
+	. "github.com/onsi/ginkgo/v2"
+	//nolint:revive // dot imports are fine for Ginkgo
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Bank Extension -", func() {
+	var (
+		// BankCallerContractAddr         common.Address
```

Comment:
Is this commented line relevant?

Reply:
Not sure if it's worth creating a caller contract for this precompile.